### PR TITLE
Import the Prawn images needed for tests and manual

### PR DESCRIPTION
**Pull request by fidalgo**
Since the Prawn team, due Gem's best practices of not including nothing
that it's not required at runtime.
An issue was opened in the project at
https://github.com/prawnpdf/prawn/issues/1041 to ask for clarification
on this and as result we should have what we need in our repo and not
relying other gems.

So in this commit we've added the needed images and replaced their use
in code and manual to make use of the local ones.

NOTE: The tests are failing due the outdated Ruby versions on Travis configuration. I will open a new PR for syncing the travis configuration with the Prawn.

This PR along with #95 will make the tests pass again.

### CODE REVIEW

File: manual/table/content_and_subtables.rb [hunk 1]
```diff
@@ -19,17 +19,16 @@
 #
 require File.expand_path(File.join(File.dirname(__FILE__),
                                    %w[.. example_helper]))
-
 filename = File.basename(__FILE__).gsub('.rb', '.pdf')
 Prawn::ManualBuilder::Example.generate(filename) do
+  image_path = File.expand_path(File.join(File.dirname(__FILE__), %w[data images stef.jpg]))
```
Comment (pointlessone) [inline manual/table/content_and_subtables.rb]: You can use `__dir__` constant.

**Comment by pointlessone** [comment]
You probably should copy over those files since they are missing from this repo.

File: manual/table/content_and_subtables.rb (hunk 1 above)
Comment (fidalgo) [reply manual/table/content_and_subtables.rb]: For some reason `__dir__` inside the manual examples is being evaluated to `nil`, so I've use it in the specs, but not here.

**Comment by fidalgo** [comment]
@pointlessone you're right, I've missed from the commit.

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,9 @@ before_install:
...
 rvm:
-  - 2.0.0-p0
-  - 2.0.0-p648
-  - 2.1.0-p0
-  - 2.1.8
-  - 2.2.0-p0
-  - 2.2.4
+  - 2.0.0
+  - 2.1.0
+  - 2.2.0
```
Comment (pointlessone) [inline .travis.yml]: This appears to be unrelated to the PR.

The versions were intentional. The idea was to run on the first and the latest release of each minor version.

Comment (fidalgo) [reply .travis.yml]: But if you see the Travis logs, it's failing on the `2.0.0-p0` version. I would try to create another PR with those changes. Also would make sense to test for versions that don't have upstream support?

Comment (pointlessone) [reply .travis.yml]: Prawn doesn't support 2.0 at this point. I'd suggest this gem maintainers to match Ruby supported version to those of Prawn but ultimately it's up to them.

I was just pointing out that new matrix doesn't cover all versions initially intended. It might be OK if the maintainers decide so.

**Comment by fidalgo** [comment]
I've updated the PR, removing the changes on Travis configuration. Also added a note on PR explaining why the tests are failing. Since this is a catch 22, I would suggest to merge this PR and then merge the #95 to update the supported Ruby versions in order to have the tests passing again.
